sorbet/rbi/shims/active_model.rbi

```diff
@@ -0,0 +1,15 @@
+# typed: true
+
+module ActiveModel
+  module Attributes
+    module ClassMethods
+      def attribute_types; end
+    end
+  end
+
+  module AttributeMethods
+    module ClassMethods
+      def attribute_method_matchers; end
```

Comment:
Still having trouble getting Sorbet to recognize this one -- I know that `ActiveModel::Attributes` [includes `ActiveModel::AttributeMethods`](https://github.com/rails/rails/blob/main/activemodel/lib/active_model/attributes.rb#L9), so I was wondering if maybe I could leverage `mixes_in_class_methods(::ActiveModel::AttributeMethods::ClassMethods)` on `ActiveModel::Attributes` somehow in addition to declaring `attribute_method_matchers` in the shim, but no luck so far 😅